prep-html-syntax/index.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
+<html>
+<h1 style="color: orange;">My favorite Star Wars Character is Baby Yoda</h1>
+<title>HTML Syntax</title>
```

Comment:
Be sure to put your `<title>` instead of a `<head>` element. The `<h1>` should go inside of the `<body>`. HTML can sort of figure out what you meant to do, but it's best to separate the visible content from the metadata.

You can check your HTML skeleton for reference.